src/main/java/com/altissia/clickandrun/domain/spreadsheet/concrete/registration/RegistrantRow.java

```diff
@@ -0,0 +1,75 @@
+package com.altissia.clickandrun.domain.spreadsheet.concrete.registration;
+
+import com.altissia.clickandrun.domain.spreadsheet.Row;
+import com.poiji.annotation.ExcelCellName;
+import com.poiji.annotation.ExcelRow;
+import org.hibernate.validator.constraints.Email;
+import org.hibernate.validator.constraints.Length;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.Objects;
+
+public class RegistrantRow extends Row {
```

Comment:
Because Registration is more like the whole process. Registration include the licenses the user would want and it's in the other sheet. Registrant is the guy who wants to go through registration, I thought it was more suitable.

Reply:
And the sheet itself is call "registrants" so, indeed it's fine.